lib/inline_svg.rb

```diff
@@ -36,7 +36,7 @@ def add_custom_transformation(options)
       if incompatible_transformation?(options.fetch(:transform))
         raise InlineSvg::Configuration::Invalid.new("#{options.fetch(:transform)} should implement the .create_with_value and #transform methods")
       end
-      @custom_transformations.merge!(Hash[ *[options.fetch(:attribute, :no_attribute), options.fetch(:transform, no_transform)] ])
+      @custom_transformations.merge!(Hash[ *[options.fetch(:attribute, :no_attribute), options] ])
```

Comment:
We end up with the entire options hash as the value here. Do you think we should remove the `attribute` key/value pair?

```ruby
@custom_transformations.merge!(Hash[ *[options.fetch(:attribute, :no_attribute), options.select { |k,_| k != :attribute } ] ])
```